Chargebee/Classes/Purchase/CBCustomer.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  CBCustomer.swift
+//  Chargebee
+//
+//  Created by ramesh_g on 06/02/23.
+//
+
+import Foundation
+
+
+public struct CBCustomer{
+    public var customerID: String
+    public var first_name: String
+    public var last_name: String
+    public var email: String
```

Comment:
Please keep the naming consistent and camelcase.

Reply:
Updated